src/main/java/uk/gov/justice/probation/courtcaseservice/service/UserAgnosticOffenderService.java

```diff
@@ -49,11 +49,12 @@ public Optional<OffenderEntity> updateOffenderProbationStatus(String crn) {
     }
 
     private OffenderEntity updateProbationStatusDetails(ProbationStatusDetail probationStatusDetail, OffenderEntity offender) {
-        return offender.withProbationStatus(OffenderProbationStatus.of(probationStatusDetail.getStatus()))
-                .withPreviouslyKnownTerminationDate(probationStatusDetail.getPreviouslyKnownTerminationDate())
-                .withBreach(probationStatusDetail.getInBreach())
-                .withAwaitingPsr(probationStatusDetail.getAwaitingPsr())
-                .withPreSentenceActivity(probationStatusDetail.isPreSentenceActivity());
+        offender.setProbationStatus(OffenderProbationStatus.of(probationStatusDetail.getStatus()));
```

Comment:
I remember mentioning this during the mob review session. I encountered similar issue during this but was caught in integration test https://github.com/ministryofjustice/court-case-service/blob/1c155006d7184f0a5635b1e67059e9b8365723df/src/main/java/uk/gov/justice/probation/courtcaseservice/jpa/repository/OffenderRepositoryFacade.java#L25-L43